tests/samplers_tests/test_samplers.py

```diff
@@ -144,17 +141,14 @@ def test_random_sampler_reseed_rng() -> None:
 @pytest.mark.parametrize(
     "distribution",
     [
-        UniformDistribution(-1.0, 1.0),
-        UniformDistribution(0.0, 1.0),
-        UniformDistribution(-1.0, 0.0),
         FloatDistribution(-1.0, 1.0),
         FloatDistribution(0.0, 1.0),
         FloatDistribution(-1.0, 0.0),
     ],
 )
 def test_uniform(
```

Comment:
It beyonds this PR but I noticed these tests check if samples exist in `[low, upper)`, which doesn't match their documentations. 👀 

https://optuna.readthedocs.io/en/stable/reference/generated/optuna.distributions.UniformDistribution.html#optuna.distributions.UniformDistribution.high